docs/README.md

```diff
@@ -258,6 +258,9 @@ cd ~/codechecker
 make venv
 source $PWD/venv/bin/activate
 
+# Optional step in case you want LDAP or PAM authentication support.
+#pip3 install -r web/requirements_py/auth/requirements.txt
+
```

Comment:
Very strong disagree here. These additional requirements usually depend on system packages which may or may not be available on the system, especially not if the user is building a container! We should not have first-time users think too much about what to do in the build process. Lack of system-level packages, like the necessary library headers, will result in building and extracting the pip package not succeed.

The documentation (https://github.com/Ericsson/codechecker/blob/master/docs/web/authentication.md#external-authentication-methods-) which you need to read to set LDAP up anyway, discusses the need for additional **system-level** packages.

Reply:
It's a real problem however. I am the same person that at some point enabled LDAP auth, building CodeChecker with the needed requirements, and at some other point needed to upgrade CodeChecker so I just followed the build instructions on the "install" page, and then people couldn't login. The problem is solved for me (by following my notes, instead of the project's install instructions) but it was subtle so I created this pull request.

I could suggest other alternatives:
* Add a bullet point noting that you might have to install more packages depending the features you want to enable, with links to the relevant pages (which are not easy to find BTW). Are there more optional features that require certain packages? It would be nice to collect this info on the installation page.
* Fail to build the server if the build configuration has LDAP enabled. I am not sure how this would be possible in CodeChecker, but other projects would do that during `./configure --enable-ldap`, for example.
* Fail to run the server if the runtime configuration has LDAP enabled.

Reply:
The pull request has changed. Do we "resolve" this conversation?